ber.go

```diff
@@ -5,7 +5,7 @@ import (
 	"errors"
 )
 
-var encodeIndent = 0
+// var encodeIndent = 0
```

Comment:
Obviously - commented out code is not good ... but I see it's all from upstream, so it's okay.